strawberry/http.py

```diff
@@ -10,12 +10,15 @@
 class GraphQLHTTPResponse(TypedDict, total=False):
     data: Optional[Dict[str, Any]]
     errors: Optional[List[Any]]
+    extensions: Optional[Dict[str, Any]]
```

Comment:
This is marked `Optional` but can never be `None`. Should we be setting `data["extensions"] = None` if there aren't any? (this would apply to `data` and `errors` too.)

Reply:
`data` and `errors` already work like that, for extensions we always return a dict: https://github.com/strawberry-graphql/strawberry/blob/98b4e6728251bfba3fa59b35a65f348f273b2e3b/strawberry/extensions/runner.py#L50-L56

I think we can keep them as optional, I'm not sure if we will change the dict to None at somepoint (for example if there's no extension). So I'd say this is fine :)

We can always update this in another PR :)